scripts/rust-boml-perf/src/main.rs

```diff
@@ -0,0 +1,11 @@
+fn main() {
+    let mut args = std::env::args_os();
+    args.next();
+    let path = std::path::PathBuf::from(args.next().expect("Missing TOML path argument"));
+    let raw = std::fs::read_to_string(&path).unwrap();
+    let timer = std::time::Instant::now();
+    let value = boml::Toml::parse(&raw);
```

Comment:
iiuc for parsers that lazily parse / produce output, you don't "force" them to produce output for the benchmarks, right?

Reply:
I don't have a clear policy on this; the main reason the benchmarks exist is to validate it's not horribly slow. That's also why I replaced the benchmark numbers with text descriptions a while ago: it's not a game who is fastest. Parsing a few numbers or dates usually isn't the problem: it's usually either the lexing that's somehow very slow, and/or performance is O(zomg) to the key size.